tensorflow/c/experimental/stream_executor/stream_executor_internal.h

```diff
@@ -37,6 +38,13 @@ port::Status InitStreamExecutorPlugin(void* dso_handle);
 // testing).
 port::Status InitStreamExecutorPlugin(SEInitPluginFn init_fn);
 
+namespace {
+struct TFStatusDeleter {
+  void operator()(TF_Status* s) const { TF_DeleteStatus(s); }
+};
+using OwnedTFStatus = std::unique_ptr<TF_Status, TFStatusDeleter>;
+}  // namespace
+
 class CPlatform : public Platform {
```

Comment:
Should we have a high level comment on what these `CXyz` classes do?

Reply:
These class are discussed in [StreamExecutor RFC](https://github.com/annarev/community/blob/21be6e9065d5c3d8bc811be3f8e887e87276931f/rfcs/20200612-stream-executor-c-api.md#stream--timer--event-representation), they are customized implementations of stream_exeuctor::StreamInterface, stream_executor::TimerInterface and stream_executor::EventInterface , which wraps those stream exeuctor C API. 
@annarev  do you have any suggestion? thanks.

Reply:
May be we can add a comment like this one to `stream_executor_internal.h`:
https://cs.opensource.google/tensorflow/tensorflow/+/master:tensorflow/c/experimental/stream_executor/stream_executor.cc;l=15;drc=12c7ef6becee3f91bf1b7be4e1f24f06f4c79041

Reply:
done